src/Compilers/CSharp/Portable/xlf/CSharpResources.cs.xlf

```diff
@@ -2552,6 +2552,11 @@
         <target state="translated">přepsání tvůrce asynchronní metody</target>
         <note />
       </trans-unit>
+      <trans-unit id="IDS_CollectionExpression">
+        <source>collection expression</source>
+        <target state="new">collection expression</target>
+        <note />
+      </trans-unit>
```

Comment:
```suggestion
      <trans-unit id="IDS_CollectionExpression">
        <source>collection expression</source>
        <target state="new">collection expression</target>
        <note />
      </trans-unit>
```

Do I need to do anything for translation? Is it translated automatically?

Reply:
> Do I need to do anything for translation?

No. Translation is handled separately